src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -2418,6 +2418,12 @@ private void RegisterHomeWorkspace(HomeWorkspaceModel newWorkspace)
         /// </summary>
         protected void SaveBackupFiles(object state)
         {
+            //No backup files in ServiceMode due to Lambda restrictions
+            if (IsServiceMode)
```

Comment:
would be nice to have a test for this if there are already backup tests

Reply:
I think we do